docs/rules/function-paren-newline.md

```diff
@@ -0,0 +1,277 @@
+# enforce consistent line breaks inside function parentheses (function-paren-newline)
+
+Many styleguides require or disallow newlines inside of function parentheses.
+
+## Rule Details
+
+This rule enforces consistent line breaks inside parentheses of function parameters or arguments.
+
+### Options
+
+This rule has a single option, which can either be a string or an object.
+
+* `"always"` requires line breaks inside all function parentheses.
```

Comment:
Which one of those is default (if any)? We should not it in the documentation, if there's one.